lib/lighthouse.js

```diff
@@ -56,15 +62,29 @@ function run() {
 
   return executeSequentially(tasks)
     .then(() => {
-      const output = reportBuilder.build();
-      fs.writeFileSync(__dirname + '/../lighthouse-report.xml', output);
-      logger.log(output);
+      const outputXml = reportBuilder.build();
+      fs.writeFileSync(__dirname + '/../lighthouse-report.xml', outputXml);
+      const outputJson = JSON.stringify(jsonOutput);
+      const jsonPath = getOutputJson();
+      fs.writeFileSync(__dirname + jsonPath, outputJson);
+      logger.log(outputXml);
     });
 }
 
-function runTestsForUrl({ url, shouldSignIn, hide }) {
+function getScoresPerCategory(reportCategories = [], url) {
+  const results = reportCategories.map((category) => {
```

Comment:
This seems to be just copying the same thing across to the new value. Is this so that you only output the name and the score and nothing else?

Also, you could use destructuring and object shorthand to make this a little tidier:
```js
  const results = reportCategories.map(({ name, score }) => ({ name, score }));
```

Reply:
Yes that is the reason for doing this. Have tidied up with destructuring.